scripts/pull_github_pr.sh

```diff
@@ -61,8 +61,13 @@ if [[ $nr_commits == 1 ]]; then
 	message="$(git log -1 "$commit" --format="format:%s%n%n%b")"
 	if ! git cherry-pick $commit
 	then
-		git cherry-pick --abort
-		exit 1
+		echo "Cherry-pick failed. You are now in a subshell. Either resolve with git cherry-pick --continue or git cherry-pick --abort, then exit the subshell"
+		head_before=$(git rev-parse HEAD)
+		bash
```

Comment:
silly nitpick: not everyone uses bash (I use zsh). You should have run `$SHELL` or ${SHELL-bash}.
But I guess I'll survive :-)